deploy/crds/storage.dell.com.crds.all.yaml

```diff
@@ -34,40 +34,55 @@ spec:
       name: v1
       schema:
         openAPIV3Schema:
-          description:
+          description: >-
```

Comment:
Is this new syntax with `>-`? Was it generated from the operator SDK?

Reply:
I see this file was copied from `./config` to here and it has those same syntax updates.